### ISSUE TITLE
Add fmin method?

### ISSUE THREAD

**Issue by MichaelClerx** [pints-team/pints#378]
We could add an `fmin` method to quickly minimise a function with Pints, e.g.:

```
def fmin(f, time, signal, p0, method=pints.CMAES):
    d = len(p0)
    class Err(pints.ErrorMeasure):
        def n_parameters(self):
            return d
        def __call__(self, parameters):
            return np.sum((signal - f(time, *parameters))**2)
    opt = pints.Optimisation(Err(), p0, method=method)
    opt.set_log_to_screen(None)
    xbest, fbest = opt.run()
    return xbest
```

Any reasons we shouldn't do this?

**Comment by MichaelClerx** [comment]
Convenience! That's all :-)

**Comment by MichaelClerx** [comment]
Plus it lets you optimise stuff outside of the context of time series quite easily... So that might be good or bad...

**Comment by martinjrobins** [comment]
Well, seems fairly inoffensive to me :) Happy to add this in

**Comment by MichaelClerx** [comment]
Cool! Might actually call it `curve_fit` instead https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.curve_fit.html
And do a separate fmin to just minimise something https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fmin.html

**Comment by MichaelClerx** [comment]
Closed in #379
